app/commands/v2/put_content.rb

```diff
@@ -168,8 +168,12 @@ def send_downstream(content_item)
         return unless downstream
 
         ContentStorePayloadVersion.increment(content_item.id)
-        ContentStoreWorker.perform_in(
-          1.second,
+
+        message = "Enqueuing ContentStoreWorker job with "
+        message += "{ content_store: Adapters::DraftContentStore, content_item_id: #{content_item.id} }"
+        logger.info message
```

Comment:
It should, and I think this logging should be moved to `perform_with_defaults` to log anything going onto the queue.